libr/anal/block.c

```diff
@@ -59,6 +60,7 @@ static RAnalBlock *block_new(RAnal *a, ut64 addr, ut64 size) {
 	block->parent_stackptr = INT_MAX;
 	block->cmpval = UT64_MAX;
 	block->fcns = r_list_new ();
+	r_anal_block_update_hash (block);
```

Comment:
uhm, ther's no data read at this point.. so not sure what should be updating here